config/etc/zinit/init/ntpd-setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+ntp_flags=$(cat /proc/cmdline | grep -o 'ntp=.*')
+
+if [ -n "$ntp_flags" ]; then
+  ntp_flags=${ntp_flags#*=}
+
+  temp_file=$(mktemp)
+
+  for server in $(echo "$ntp_flags" | tr ',' ' '); do
+    echo "server $server" >> $temp_file
```

Comment:
Why do you need a temp file, can't you just write the content directly to the ntp.conf file?

Reply:
you first clear the file (either remove, or truncate) then in this loop just do the echo directly to the config file.

Reply:
Or you can just append `-p PEER` (can be repeated) :p

Reply:
`exec ntpd -n $(echo "-p $ntp_flags" | sed s/,/' -p '/g)`
Should just works.

Reply:
- > Or you can just append -p PEER (can be repeated) :p

  I checked the `ntpd` docs and looks `-p` flag is for providing `pidfile` not a peer server
  ![image](https://github.com/threefoldtech/0-initramfs/assets/32527347/cef2d6eb-9d90-486f-9b36-2ba23d3dc09e)


- > you first clear the file (either remove, or truncate) then in this loop just do the echo directly to the config file.
  
  i didn't want to clear the file until i am sure there is content that will override the existing one. providing an empty flag like `ntp=` will empty the file with no new content. but yeah not the best approach.

  i modified the script with a better check for the content of `ntp_flags` and got rid of the tmp file.

Reply:
Which ntpd doc did you read ? We use the ntpd from busybox.

Reply:
@maxux 
I think using the `-p` flag is a good idea. And then we don't need to even update the file. passing a comma separated list of peers is also a good touch. Thank you

Reply:
> Which ntpd doc did you read ? We use the ntpd from busybox.

i think i was reading another ntpd docs. checked the one on zos and it do use `-p` for peers

updated the code to use -p as suggested